docs/integrations/airbyte.md

```diff
@@ -1,53 +1,53 @@
-to warehouses, lakes and other destinations. Using Airbyte's connectors you can get your data pipelines running
-to consolidate many input sources.
+[Airbyte](https://airbyte.io//) is an open-source platform to sync data from applications, APIs & databases to
+warehouses, lakes and other destinations.  Use Airbyte's connectors to get your data pipelines to consolidate
+many input sources.
 
 ## Table of contents
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc .pb-5 }
-   
+
 ## Using lakeFS with Airbyte
 The integration between the two open-source projects brings resilience and manageability when using Airbyte
 connectors to sync data to your S3 buckets by leveraging lakeFS branches and atomic commits and merges.
 
 ## Use-cases
-You can leverage lakeFS ACID guarantees and [CI/CD capabilities](../usecases/ci.md) when ingesting data to S3 using lakeFS:
+You can leverage lakeFS consistency guarantees and [CI/CD capabilities](../usecases/ci.md) when ingesting data to S3 using lakeFS:
 
-1. Consolidate many data source into a single branch and expose them to the consumers simultaneously when merging to the `main` branch.
-1. Test the incoming data for breaking schema changes, using [lakeFS hooks](../setup/hooks.md). 
-1. Avoid having consumers reading partial data from connectors which failed half-way through the sync operation.
-1. Experiment with ingested data before exposing it.
+1. Consolidate many data sources to a single branch and expose them to the consumers simultaneously when merging to the `main` branch.
+1. Test incoming data for breaking schema changes, using [lakeFS hooks](../setup/hooks.md).
+1. Prevent consumers from reading partial data from connectors which failed half-way through sync.
+1. Experiment with ingested data on a branch before exposing it.
 
 ## S3 Connector
 lakeFS exposes an [S3 Gateway](../understand/architecture.md#s3-gateway) that enables applications to communicate
-with lakeFS the same way they would with Amazon S3.
+with lakeFS in the same way they would with Amazon S3.
 You can use Airbyte's [S3 Destination](https://airbyte.io/destinations/s3) for uploading the data to lakeFS.
 
 ### Configuring lakeFS using the connector
 Set the following parameters when creating a new Destination of type S3:
 
 | Name             | Value                                                        | Example                                                                                                             |
 |------------------|--------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
-| Endpoint         | The lakeFS S3 gateway URL                                    | `http://lakefs.example.com`                                                                                       |
+| Endpoint         | The lakeFS S3 gateway URL                                    | `https://cute-axolotol.lakefs-demo.io`
```

Comment:
Wondering if our docs should use lakefs Playground URLs ase examples.

Reply:
I (obviously) think that they should.

But I agree this is an unrelated change and should not block the other fixes.  Let me know if you would prefer me to drop this change -- perhaps we can/should discuss it separately